mystery-deckbuilder/Assets/Scripts/StateControl/GameState.cs

```diff
@@ -49,6 +50,17 @@ public class NPCs
 
     }
 
+    /* GameState holder class for ongoing card and deck information*/
+    public class CardInfo
+    {
+        //these are lists because they have to be in this context. just be mindfull of list length weirdness
+        static int[] startingDeck = {1, 5, 9, 1, 5, 9, 1, 5, 9, 1, 5, 9};
+        public static GameStateValue<List<int>> currentDeck = new(new List<int>(startingDeck), _gameStateValues);
+
+        static int[] startingDiscard = { };
+        public static GameStateValue<List<int>> currentDiscard = new(new List<int>(startingDiscard), _gameStateValues);
```

Comment:
NIT: could be a literal, and could use startingDiscard.ToList() or whatever it is called